en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,184 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page data from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url. Recent browser (such as ``Chrome`` or ``Firefox`` can perform
+up to 10 simultaneous requests). In the following documentation, we would just assume that
+browsers can make only two simultaneous request to the same domain.
+
+An easy way to display would be to add a new virtual host to your existing Open Orchestra
+installation. This will allow you to make two parallel requests to get the page (with the
+javascript and the css) and two parallel requests to the media server (to get all the images).
```

Comment:
to avoid couting requests in your example (2 request to medias, 2 request to webserver, etc.) I would rephrase to something with the following idea "This will allow your browser to increase the number of parallel requests that can be done as several domain are available. It will then reduce the global latency of each page's download."